spring/boot-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaReactiveWebServerFactory.java

```diff
@@ -108,7 +134,22 @@ public WebServer getWebServer(HttpHandler httpHandler) {
         final Ssl ssl = getSsl();
         if (ssl != null) {
             if (ssl.isEnabled()) {
-                configureTls(sb, ssl, getSslStoreProvider());
+                final SslStoreProvider provider = getSslStoreProvider();
+                configureTls(sb, toArmeriaSslConfiguration(ssl),
+                             () -> {
```

Comment:
I think that if the `provider` is `null`, this supplier must be `null`. So this kind of fix might be necessary:
```
final Supplier<KeyStore> keyStoreSupplier;
final Supplier<KeyStore> trustStoreSupplier;
if (provider != null) {
    keyStoreSupplier = () -> {
        try {
            return provider.getKeyStore();
        } catch (Exception e) {
            throw new IllegalStateException(e);
        }
    };
    trustStoreSupplier = () -> {
        try {
            return provider.getTrustStore();
        } catch (Exception e) {
            throw new IllegalStateException(e);
        }
    };
} else {
    keyStoreSupplier = null;
    trustStoreSupplier = null;
}

configureTls(sb, toArmeriaSslConfiguration(ssl), keyStoreSupplier, trustStoreSupplier);
```